doc/self-hosted/index.md

```diff
@@ -50,10 +50,27 @@ Options:
   -p, --port <number>         Port to bind on. (default: 8443)
   -N, --no-auth               Start without requiring authentication.
   -H, --allow-http            Allow http connections.
-  -P, --password <value>      Specify a password for authentication.
+  -P, --password <value>      DEPRECATED in 1.903-vsc1.33.1: Use "PASSWORD" environment variable instead. Specify a password for authentication. 
```

Comment:
Since this is the output of the `--help` flag, I'm thinking it should directly copy that text.

Reply:
Valid. I’ll make those changes tonight.

Reply:
Valid. I’ll make those changes tonight.